modules/ROOT/pages/audit-logging.adoc

```diff
@@ -2099,13 +2099,29 @@ You can access the
 https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/audit-log-query-api/[Audit Log REST API] from the Audit Logging Query API and its
 https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/organizations/8bfc8bbf-5508-419e-aadc-77dfe18a8172/assets/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/audit-log-query-api/1.0.5/files/fat-raml/zip[RAML].
 
+
 === Use a Query Loop to make Requests in Audit Logs
 
-To gather data, you can initiate query loops and use cursors to navigate paginated audit logs. To avoid performance degradation, you can stop the query loop using the `total` by using the value returned by `IncludeTotal`.
+To query audit logs, the current https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/audit-log-query-api/minor/1.0/console/method/%23245/ [query api] will continue to work.
+Pagination in that api is via offset in the query payload, and the response returns the total number of records.
+
+
+We are adding a new pagination using cursor and you can initiate query loops by using cursors. This cursor paging can be used by using the `cursorPagination` query parameter and setting it to true.
+When using cursor paging, each response will return a cursor. This cursor corresponds to the last data entry in the response data set. So the next request in query loop should include the cursor value that was returned in the previous response, and so on.
+When the data set is empty in response, then you have reached the end of query loop, and there is no more data to query for that time window. So, that is the indicator the stop the query loop. When the data set is empty in the response, there will be no cursor returned.
+Total is not returned by default when using cursor pagination, as the query loop is not dependent on offset and number of total rows.
```

Comment:
```suggestion
When you use cursor pagination, `total` is not returned by default, because the query loop is not dependent on the offset or the number of total rows.
```